examples/hello_world/build.rs

```diff
@@ -1,11 +1,12 @@
-extern crate protoc_rust;
+extern crate protoc_rust_grpc;
 
 fn main() {
-    protoc_rust::run(protoc_rust::Args {
+    protoc_rust_grpc::run(protoc_rust_grpc::Args {
         out_dir: "src/proto",
         input: &["proto/hello_world.proto"],
         includes: &[],
-        customize: protoc_rust::Customize::default(),
+        rust_protobuf: true, // also generate protobuf messages, not just services
```

Comment:
Why not use customize with a new type? We are arguably generating a customised version.

Reply:
I don't think the `customize` [option](https://docs.rs/protoc-rust/2.6.2/protoc_rust/struct.Customize.html) to the base `protoc_rust` tool covers enough ground to allow generation of per-service boilerplate &ndash; it seems to mostly govern accessor generation etc. for the per-message `struct`s.